tests/molecular/topology_graphs/conftest.py

```diff
@@ -22,15 +22,51 @@
             ),
         ),
         CaseData(
-            topology_graph=stk.cof.Honeycomb(
+            topology_graph=stk.cof.PeriodicHoneycomb(
+                building_blocks=(bb1, bb2),
+                lattice_size=(3, 1, 2),
+                optimizer=stk.PeriodicCollapser(),
+            ),
+            cell=(
+                np.array([109.29499828, 0., 0.]),
+                np.array([18.21583305, 31.54982284, 0.]),
+                np.array([0., 0., 210.33234855])
+            ),
+        ),
+    ),
+)
+def unscaled_periodic_case(request):
```

Comment:
I think this is a great example of when an inline comment would be helpful - the developer has no idea, without looking at the test, why this would be an unscaled test case...

Reply:
Good catch. Yeah these test cases are meant to avoid using optimizers which edit the `PeriodicInfo`. I think putting it into the docstring would work too, as it's essentially part of the interface. Unless you have a particular reason why you prefer inline for this.